variables.tf

```
@@ -40,6 +40,22 @@ variable "admin_user" {
   default     = "postgres"
 }
 
+variable "resource_proxy" {
+  type = object({
+    cpu     = number,
+    memory  = number
+  })
+  description = "Postgres proxy resources"
+  default = {
+    cpu         = 500
+    memory      = 1024
+  }
+  validation {
+    condition     = var.resource_proxy.cpu >= 500 && var.resource_proxy.memory >= 1024
+    error_message = "Proxy resource must be at least: cpu=500, memory=1024."
+  }
+}
```

The minimal should be like this:
```suggestion
variable "resource_proxy" {
  type = object({
    cpu     = number,
    memory  = number
  })
  description = "Postgres proxy resources"
  default = {
    cpu         = 200
    memory      = 128
  }
  validation {
    condition     = var.resource_proxy.cpu >= 200 && var.resource_proxy.memory >= 128
    error_message = "Proxy resource must be at least: cpu=200, memory=128."
  }
}
```